nhais/outbound/request/handler.py

```diff
@@ -26,6 +28,13 @@ def __init__(self, application: "Application", request: httputil.HTTPServerReque
 
     @timing.time_request
     async def post(self):
-        edifact = self.fhir_to_edifact.convert(self.request.body.decode())
-        await self.mesh_wrapper.send(edifact)
-        self.set_status(202)
+        is_valid_schema = generate_jsonschema.validate_schema(self.request.body.decode())
+        if(is_valid_schema):
+            edifact = self.fhir_to_edifact.convert(self.request.body.decode())
+            unique_operation_id = message_utilities.get_uuid()
+            await self.mesh_wrapper.send(edifact)
+            self.set_status(202)
+            self.finish(f"<html><body>UniqueOperationID: {unique_operation_id}</body></html>")
```

Comment:
The response needs to be a FHIR OperationOutcome with content type application/json

https://portal.developer.nhs.uk/docs/personal-demographics-dev/1/types/OperationOutcome

Something like: https://github.com/nhsconnect/integration-adaptors/blob/feature/NIAD-170-nhais-openapi-spec/nhais/specification/components/examples/responses/with-operation-id.json

Reply:
correct OperationOutcome now implemented